src/container.ts

```diff
@@ -107,16 +131,47 @@ container.register({
   apiUrl: awilix.asValue(process.env.API_URL)
 });
 
+// Notification URL pre shared key.
+if (process.env.PRE_SHARED_KEY === undefined) {
+  winston.error("Missing PRE_SHARED_KEY environment variable");
+  process.exit(0);
```

Comment:
I'd use an exit value != 0 (success). If this is the main file "return 1" should suffice I guess.